pkg/geo/geogfn/unary_operators.go

```diff
@@ -128,3 +166,32 @@ func length(regions []s2.Region, useSphereOrSpheroid UseSphereOrSpheroid) (float
 	}
 	return totalLength, nil
 }
+
+// latitudeNormalize convert a latitude to the range of -Pi/2, Pi/2.
+func latitudeNormalize(lat float64) float64 {
```

Comment:
nit: can you name this `normalizeLatitude`?
also shame there isn't something similar in s1/s2 :(

Reply:
Done.